disk-utils/cfdisk.c

```diff
@@ -22,6 +22,8 @@
 #include <libsmartcols.h>
 #include <sys/ioctl.h>
 #include <libfdisk.h>
+#include <fdiskP.h>
```

Comment:
fdiskP.h is a private libfdisk header file and you really don't want to use it applications.